### PR TITLE
fix: fix react warning React does not recognize the `noStyle` prop on…

### PR DESCRIPTION
**Pull request by fatelei**
… a DOM element

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/table/Table.tsx [hunk 1]
```diff
@@ -944,7 +944,7 @@ export default class Table<T> extends React.Component<TableProps<T>, TableState<
                 onExpand(record, event);
               }}
               aria-label={expanded ? locale.collapse : locale.expand}
-              noStyle
+              noStyle={true}
```
Comment (ztplz) [inline components/table/Table.tsx]: 你这么改还是会错的 table里是把属性都传下去了  但是dom没有nostyle这个属性

**Comment by ztplz** [comment]
我再看看

**Comment by ztplz** [comment]
https://github.com/ant-design/ant-design/commit/9630410b4e955316f890521ecb80f3a44d21a53e
已经修复了

**Comment by fatelei** [comment]
好的

**Pull request closed without merging**